### PR TITLE
Setup CI for Serverless release 1.14

### PR DESCRIPTION
**Pull request by lance**
This is a duplication of @markusthoemmes https://github.com/openshift/release/pull/15122 for the Serverless 1.14 release. Markus, can you please review?

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.7.yaml [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+          cpu: 100m
+          memory: 200Mi
+    workflow: openshift-e2e-aws
+- as: e2e-aws-ocp-47-continuous
+  cron: 0 */12 * * 1-5
+  steps:
+    cluster_profile: aws
+    test:
+    - as: serverless-e2e
+      commands: make test-e2e test-upstream-e2e-no-upgrade
```
Comment (mgencur) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.7.yaml:162]:
```suggestion
      commands: make test-e2e-with-kafka test-upstream-e2e-no-upgrade
```

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.6.yaml [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+          cpu: 100m
+          memory: 200Mi
+    workflow: openshift-e2e-aws
+- as: e2e-aws-ocp-46-continuous
+  cron: 0 */12 * * 1-5
+  steps:
+    cluster_profile: aws
+    test:
+    - as: serverless-e2e
+      commands: make test-e2e test-upstream-e2e-no-upgrade
```
Comment (mgencur) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.6.yaml:160]:
```suggestion
      commands: make test-e2e-with-kafka test-upstream-e2e-no-upgrade
```

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.7.yaml [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+promotion:
+  disabled: true
+  name: openshift-serverless-v1.14.0
+  namespace: openshift
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.7"
```
Comment (mgencur) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.7.yaml:103]: The above should be changed to:
```
releases:
  latest:
    release:
      channel: stable
      version: "4.7"
```
The stable version is out. We don't need to test with a nightly build.

**Review by mgencur — CHANGES_REQUESTED** [review]
Hey @lance , just a few issues. Otherwise looks good. Thanks

**Comment by mgencur** [comment]
I've put lgtm here to unblock CI for release-1.14. The issues are minor and I'll send a fix for them in a follow-up PR.

**Comment by mgencur** [comment]
The follow-up PR is here: https://github.com/openshift/release/pull/17489

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.6.yaml [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+- context_dir: openshift/ci-operator/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    openshift_knative-v0.19.0_knative-serving-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.19.0:knative-serving-src
```
Comment (dsimansk) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.6.yaml:71]: @mgencur I believe these image references should be update to `v0.20.0` and added to base images section as well.

Comment (mgencur) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.6.yaml:71]: This is done automatically. And it was already done here one hour ago: https://github.com/openshift/release/commit/6a37b42bc822f5add417d26236248586b1556b79

Comment (dsimansk) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.6.yaml:71]: That's neat and good to know, thanks! :)

**Pull request merged**